assets/src/dashboard/app/reducer/settings.js

```diff
@@ -69,6 +77,27 @@ function settingsReducer(state, action) {
       };
     }
 
+    case ACTION_TYPES.UPDATE_SETTINGS_SUCCESS: {
+      return {
+        ...state,
+        activePublisherLogoId: action.payload.activePublisherLogoId,
+        error: {},
+        settingSaved: true,
+        googleAnalyticsId: action.payload.googleAnalyticsId,
+        adSensePublisherId: action.payload.adSensePublisherId,
+        adSenseSlotId: action.payload.adSenseSlotId,
+        adManagerSlotId: action.payload.adManagerSlotId,
+        adNetwork: action.payload.adNetwork,
+        publisherLogoIds: [
+          ...new Set([
+            action.payload.activePublisherLogoId,
+            ...action.payload.publisherLogoIds,
+          ]),
+        ],
+        videoCache: action.payload.videoCache,
+      };
+    }
+
```

Comment:
Isn't this the same as `ACTION_TYPES.FETCH_SETTINGS_SUCCESS`?

Reply:
save for the `settingSaved` - lemme rework this, it's rather cluttery.

Reply:
ok it should be much better 😊